Makefile

```diff
@@ -17,17 +17,13 @@ distclean: clean
 
 .PHONY: lint
 lint:
-	$(GOLANGCILINT) run
-
-.PHONY: dep
-dep:
-	$(DEP) ensure --vendor-only
+	GO111MODULE=on $(GOLANGCILINT) run
```

Comment:
If there is no vendor folder present, what happens when you run `make lint`? Will it download the modules?

Reply:
Had to verify that one. It works as expected, downloading the modules for before linting.

I tested by `rm -rf $GOPATH/pkg/*` then `./configure && make lint`.